integrations/ScriptLoader.js

```diff
@@ -13,6 +14,7 @@ const ScriptLoader = (id, src, async = defaultAsyncState) => {
   js.type = "text/javascript";
   js.id = id;
   js.dataset.loader = LOAD_ORIGIN;
+  js.dataset.isNative = IS_NATIVE_SCRIPT;
```

Comment:
Why are you setting this by default? Now, any script loaded via this utility will become a native script.

Reply:
In the v1 version, we only load native scripts.
For v1.1 I will add logic here.

Reply:
As an example to counter that argument, we load polyfill script in both versions. Hence, let's keep the implementation the same across.